worker/uniter/uniter.go

```diff
@@ -118,12 +118,18 @@ type Uniter struct {
 	downloader charm.Downloader
 }
 
+// LeadershipTrackerWorker represents a leadership tracker worker.
+type LeadershipTrackerWorker interface {
```

Comment:
don't need this struct

The type we should use is *worker.leadership.Tracker which is both a Worker and a leadership.Tracker

Reply:
just moved `TrackerWorker` `Interface` from uniter to `core/leadership` package.
so `TrackerWorker` combines `Tracker` & `Worker` because we need Worker interface to `Init` it in catacomb. 
`worker.leadership.Tracker` is a `struct` but we'd better to use an interface here?